### ISSUE TITLE
CreditCardDirectivesModule is not an NgModule

### ISSUE THREAD

**Issue by osazemeu** [nogorilla/angular-cc-library#9]
```
MacOS Sierra
version 10.12.4
npm v4.1.2

angular-cli: 1.0.0-beta.28.3
node: 6.9.5
os: darwin x64
@angular/common: 2.4.10
@angular/compiler: 2.4.10
@angular/core: 2.4.10
@angular/forms: 2.4.10
@angular/http: 2.4.10
@angular/platform-browser: 2.4.10
@angular/platform-browser-dynamic: 2.4.10
@angular/router: 3.4.10
@angular/compiler-cli: 2.4.10

```

I get the following error on attempt to cold restart dev server.

ERROR in CreditCardDirectivesModule is not an NgModule
webpack: Failed to compile.

**Comment by jwalk830** [comment]
I'm also getting the same error.
"dependencies": {
    "@angular/common": "~4.1.0",
    "@angular/compiler": "~4.1.0",
    "@angular/compiler-cli": "~4.1.0",
    "@angular/core": "~4.1.0",
    "@angular/forms": "~4.1.0",
    "@angular/http": "~4.0.3",
    "@angular/platform-browser": "~4.1.0",
    "@angular/platform-browser-dynamic": "~4.1.0",
    "@angular/platform-server": "~4.0.2",
    "@angular/router": "~4.1.0",
    "angular-cc-library": "^1.0.3",
    "classlist.js": "~1.1.20150312",
    "core-js": "~2.4.1",
    "intl": "~1.2.5",
    "ng2-bootstrap": "~1.6.1",
    "node-testrail": "0.0.12",
    "rxjs": "~5.1.0",
    "web-animations-js": "~2.2.5",
    "zone.js": "~0.8.4"
  },
  "devDependencies": {
    "@angular/animations": "~4.1.0",
    "@angular/cli": "~1.0.1",
    "@types/jasmine": "2.5.38",
    "@types/node": "~6.0.60",
    "bootstrap-sass": "~3.3.7",
    "codelyzer": "~2.0.0",
    "jasmine-core": "~2.5.2",
    "jasmine-reporters": "~2.2.1",
    "jasmine-spec-reporter": "~3.2.0",
    "jasmine-xml2html-converter": "0.0.2",
    "karma": "~1.4.1",
    "karma-chrome-launcher": "~2.0.0",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "~0.2.0",
    "karma-istanbul-threshold": "^1.2.2",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "~0.2.2",
    "karma-phantomjs-launcher": "^1.0.4",
    "karma-spec-reporter": "0.0.31",
    "node-sass": "~4.5.1",
    "protractor": "~5.1.1",
    "ts-node": "~2.0.0",
    "tslint": "~4.5.0",
    "typescript": "~2.2.0"
  }

**Comment by maitham1** [comment]
I keep getting this issue when using ngc build

**Comment by codeman061988** [comment]
Mine is intermittent. I'm using Angular CLI to build / "serve". When I first installed and ran for the first time it worked. Second time, I received this error and the project would not compile. 
If we can get past this bug, I really think this project has some potential.

**Comment by osazemeu** [comment]
I was able to temporarily byepass this error by using the aot false flag in my production build. 
`ng build --prod --aot=false`
This is a temporal fix.

**Comment by codeman061988** [comment]
Next chance I get I will try this.

**Comment by vishal41190** [comment]
+1

**Comment by mcpadma** [comment]
Have you resolved that issue permanently??
If yes, How??
I am stuck with the same error

**Comment by jaredtapscott** [comment]
+1
I'm getting the same error.  I cannot run 'ng build' at all.

**Comment by nitishthakrarsa** [comment]
`ng build --prod --aot=false`

Even after aot false getting same error 

`ERROR in Unexpected value 'CreditCardDirectivesModule in /node_modules/angular-cc-library/dist/directives.d.ts' imported by the module 'Ap
pModule in /src/app/app.module.ts'. Please add a @NgModule annotation.`

**Comment by vmachacek** [comment]
+1

**Comment by jaysonlane** [comment]
+1

**Comment by jfox459** [comment]
+1

**Comment by netopolit** [comment]
+1

**Comment by javicebrian** [comment]
+1

**Comment by nogorilla** [comment]
closed via #15
